packages/xpath/test/native/node-name.spec.ts

```diff
@@ -84,7 +84,7 @@ describe('node name for', () => {
 	it('attribute with a specific name', () => {
 		const contextNode = document.getElementById('StepNodeTestCaseNameTestAttribute')!;
 		const attribute = getNonNamespaceAttributes(contextNode).find(
-			(attribute) => attribute.localName === 'attrib3'
+			(att) => att.localName === 'attrib3'
```

Comment:
If we're going from full word to shorthand, I'd prefer to use the more idiomatic `attr`.